tools/pipelines/templates/include-set-package-version.yml

```diff
@@ -21,6 +21,11 @@ parameters:
   type: boolean
   default: true
 
+# The semver range constraint to use for interdependencies; that is, dependencies on other packages within the release
+# group
+- name: interdependencyRange
+  type: string
```

Comment:
The other instances of this declare `^` as the default - do we not want that here?